frontend/src/containers/LoginForm/index.tsx

```diff
@@ -1,32 +1,34 @@
 import { Button, Link, TextField } from "@mui/material";
-import { useLoginMutation } from "generated/graphql";
 import { useAppDispatch, useAppSelector } from "hooks/redux";
 import { ChangeEvent, FC, FormEvent, useState } from "react";
 import { useLocation, useNavigate } from "react-router-dom";
-import { loginUser } from "store/reducers/UserReducer/ActionCreator";
+import { userAPI } from "services/userService";
+import { addUser } from "store/reducers/UserReducer/ActionCreator";
+import { ILoginRes } from "types/responses";
 
 import s from "./LoginForm.module.scss";
 
 export const LoginForm: FC = () => {
   const location = useLocation();
   const navigate = useNavigate();
   const dispatch = useAppDispatch();
-  const [ loginMutationTrigger ] = useLoginMutation();
 
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
-  const {error} = useAppSelector(state => state.userReducer);
+  const {user} = useAppSelector(state => state.userReducer);
+  const [authUser, {isError}] = userAPI.useAuthUserMutation();
 
   const fromPage = (location.state as {from: any})?.from?.pathname || '';
 
   const handleSubmit = (event: FormEvent<HTMLFormElement>) =>  {
-    dispatch(loginUser({
-      loginMutationTrigger,
-      credentials: { email, password }
-    }))
-      .then(() => {
-        debugger;
+    authUser({ identifier: email, password })
+      .then(res => {
+        localStorage.setItem('token', (res as { data: ILoginRes; }).data.jwt);
+        dispatch(addUser((res as { data: ILoginRes; }).data.user));
```

Comment:
Think about utils function or hook component, it is too much async logic for containers handler